### PR TITLE
ApiSlide to UiSlide mapping function

### PR DESCRIPTION
**Pull request by StefanoA1**
**Detailed purpose of the PR**

- Fix store setup + prepare the conditions to use the loader for the slides VIEW.

- Fix fixtures/tests linting errors (hidden because of .eslintignore match).

- Uses slideFromApi instead of a UI Slide for fetchSlide mock.

- Implements `map-api-slide-to-ui` converter fonction.

- Fixes typing.

- First iteration on the function as `choices` is not taken into FETCH_SLIDES scope.

**Result and observation**

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra lib ?**
      If checked, Which extra lib did you add ? (name, purpose, link ...).

**Testing Strategy**

- [ ] Already covered by tests
- [ ] Manual testing
- [ ] Unit testing

### CODE REVIEW

**Comment by StefanoA1** [comment]
[Add Redux action: FETCH_SLIDE](https://trello.com/c/Ng7LlU6D/2684-add-redux-action-fetchslide)

**Review by esa-coorp — APPROVED** [review]

File: packages/@coorpacademy-app-review/src/test/index.test.tsx [hunk 1]
```diff
@@ -65,24 +65,35 @@ const appOptions: AppOptions = {
...
 
-  const wrapper = container.querySelector('[data-name="loader"]');
-  t.truthy(wrapper);
-  /*
-  const wrapper = container.querySelector('[data-name="slides-revision-container"]');
-  t.truthy(wrapper);
+  const loader = container.querySelector('[data-name="loader"]');
+  t.truthy(loader);
+
+  // await waitFor(async () => {
```
Comment (StefanoA1) [inline packages/@coorpacademy-app-review/src/test/index.test.tsx:77]: commented tests until FETCH_SLIDES + the slides refactor is done

File: packages/@coorpacademy-app-review/src/types/common.ts [hunk 2]
```diff
@@ -7,14 +7,81 @@ type TemplateContextValues = {
...
+  type: 'qcmGraphic';
+};
+
+export type QcmDragQuestion = Question & {
+  type: 'qcmDrag';
+};
+
+export type BasicQuestion = Omit<Question, 'content'> & {
+  type: 'basic';
+  content: Omit<BaseContent, 'choices'> & {
```
Comment (chrisdugne) [inline packages/@coorpacademy-app-review/src/types/common.ts:47]: plutot que de faire le Omit, autant dupliquer directement `media?: unknown;` ici, ce serait plus lisible ?

File: packages/@coorpacademy-app-review/src/types/common.ts [hunk 3]
```diff
@@ -7,14 +7,81 @@ type TemplateContextValues = {
...
+    label?: string;
+    placeholder?: string;
+    id: string;
+  };
+};
+
+export type SliderQuestion = Omit<Question, 'content'> & {
+  content: Omit<BaseContent, 'choices'> & {
```
Comment (chrisdugne) [inline packages/@coorpacademy-app-review/src/types/common.ts:55]: plutot que de faire le Omit, autant dupliquer directement `media?: unknown;` ici, ce serait plus lisible ?

File: packages/@coorpacademy-app-review/src/helpers/test/fixtures/qcm-drag.ts [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+export const qcmDragSlide: SlideFromAPI = {
+  id: 'sli_419IlQQnf',
+  universalRef: 'sli_419IlQQnf',
+  question: {
+    type: 'qcmDrag',
+    header: 'sli_419IlQQnf.header',
+    explanation: 'sli_419IlQQnf.explanation',
+    content: {
+      choices: [
```
Comment (chrisdugne) [inline packages/@coorpacademy-app-review/src/helpers/test/fixtures/qcm-drag.ts:13]: ah coté API on a déjà `choices`, c'est un bon début ;)
il faudrait le faire descendre jusqu'à la vue

**Review by chrisdugne — APPROVED** [review]

**Comment by StefanoA1** [comment]
[[OR] api slides to ui slides Mapper Function](https://trello.com/c/VpQQbFfn/2691-or-api-slides-to-ui-slides-mapper-function)

**Pull request merged**